README.md

```diff
@@ -371,6 +372,22 @@ Example:
 $ sam local invoke --log-file ./output.log
 ```
 
+
+### Remote Docker
+Sam Local loads the function code using Docker Volume. As a result, The project directory must be pre-mounted on the remote host where the Docker is running.
```

Comment:
I would say "loads function code by _mounting_ filesystem to a Docker Volume". Just to make it obvious.